registration.html

```diff
@@ -0,0 +1,382 @@
+        <input type="text" size=65 name="lastname" class="form-control" id="validationCustom02" placeholder="einstein"
+          required>
+      </div>
+      <div class="col-sm-3">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-3">
+      </div>
+      <div class="col-sm-3">
+        <label for="validationCustom03">Your age in years</label>
+        <input type="text" size=65 name="yearuser" class="form-control" id="validationCustom03" placeholder="15"
+          required>
+      </div>
+      <div class="col-sm-3">
+      </div>
+      <div class="col-sm-3">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-3">
+      </div>
+      <div class="col-sm-3">
+        <label for="validationCustom04">City</label>
+        <input type="text" size=40 name="city" class="form-control" id="validationCustom04" placeholder="munich"
+          required>
+      </div>
+      <div class="col-sm-3 autocomplete">
+        <label for="validationCustom05">Country</label>
+        <input class="form-control" id="validationCustom05" type="text" name="country" placeholder="Country" required>
+      </div>
+      <div class="col-sm-3">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-3">
+      </div>
+      <div class="col-sm-3">
+        <label for="validationCustom06">Email address:</label>
+        <input type="text" size=65 name="email" class="form-control" id="validationCustom06" value="" required>
+      </div>
+      <div class="col-sm-3">
+        <label for="validationCustom07">Confirm email address:</label>
+        <input type="text" size=65 name="confirmemail" class="form-control" id="validationCustom07" value="" required>
+      </div>
+      <div class="col-sm-3">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-3">
+      </div>
+      <div class="col-sm-3">
+        <label for="validationCustom01">Password:</label>
+        <input type="text" size=65 name="passworduser" class="form-control" id="validationCustom08" value="" required>
+        <div class="form-check">
+          <input name="TAC" class="form-check-input" type="checkbox" id="gridCheck">
+          <label class="form-check-label" for="gridCheck">I agree with Terms & Conditions</label>
+          </label>
+        </div>
+        <div class="col-sm-3">
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-3">
+        </div>
+        <div class="col-sm-3">
+        </div>
+        <p></p>
+        <div class="col-sm-3">
+          <button class="btn btn-danger" style="background-color: orangered" type="submit">Create an account</button>
+        </div>
+      </div>
+      <div class="col-sm-3">
+      </div>
+    </div>
+  </form>
+  <script>
+    function autocomplete(inp, arr) {
+
+      var currentFocus;
+
+      inp.addEventListener("input", function (e) {
+        var a, b, i, val = this.value;
```

Comment:
there are bad variable names. It should be clear what something is by it's name. what is a or b? Only exception is `i` inside a for loop